python/ray/tune/tests/test_relative_local_dir.py

```diff
@@ -0,0 +1,137 @@
+# coding: utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+import unittest
+
+import ray
+from ray import tune
+from ray.rllib import _register_all
+
+
+class TuneRelativeLocalDirTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.current_dir = os.path.abspath(".")
+        ray.init(num_cpus=1, num_gpus=0, local_mode=True)
+        _register_all()
+
+    @classmethod
+    def tearDownClass(self):
+        ray.shutdown()
+
+    def tearDown(self):
+        os.chdir(self.current_dir)  # Turn the current_dir back
+
+    def testDottedRelativePath(self):
+        local_dir = "test_dotted_relative_local_dir"
+        local_dir = os.path.join(self.current_dir, local_dir)
+
```

Comment:
To make sure that the directory is actually created in the `tune.run` and also so that we don't remove any existing directories, can you add the following check here:
```suggestion
    self.assertFalse(os.path.isdir(local_dir))

```

Reply:
Sorry, this comment should've been in the `testTildeAbsolutePath` test.